python/tests/_printf.sh

```diff
@@ -0,0 +1 @@
+python _printf.py >/tmp/___printf.log 2>&1
```

Comment:
I suppose this should be something you are looking for?

```
def test_myoutput(capsys):  # or use "capfd" for fd-level
    print("hello")
    sys.stderr.write("world\n")
    captured = capsys.readouterr()
    assert captured.out == "hello\n"
    assert captured.err == "world\n"
    print("next")
    captured = capsys.readouterr()
    assert captured.out == "next\n"
```

or, maybe we could consider of using lit & FileCheck to verify the printing functionality.
and python test to only verify the correctness.

Reply:
Yes, at least we don't want to include an independent sh file

Reply:
Done.